src/pages/WalletsStart/components/NewWalletButtons/NewWalletButtons.js

```diff
@@ -31,28 +34,27 @@ export class NewWalletButtons extends PureComponent<Props> {
 
   render() {
     return (
-      <div className={`__new-wallet-buttons ${newWalletButtonsStyle.core}`}>
-        <JLink
+      <Fragment>
+        <JRaisedButton
           onClick={this.handleClickCreate}
-          className={newWalletButtonsStyle.create}
-          href='/wallets/create'
+          className={`__create-button ${newWalletButtonsStyle.create}`}
         >
           {t`Create Wallet`}
-        </JLink>
-        <span className={newWalletButtonsStyle.text}>
+        </JRaisedButton>
+        <div className={newWalletButtonsStyle.text}>
           {t`Create your own wallet to manage your digital assets`}
-        </span>
-        <JLink
+        </div>
+        <JRaisedButton
           onClick={this.handleClickImport}
-          className={newWalletButtonsStyle.import}
-          href='/wallets/import'
```

Comment:
why did you remove `href`s?

Reply:
JLink buttons could pass user to separate pages, when he wants to open them in new tab

Reply:
Reverted to the old behaviour!